cmd/dlt/cluster/cmd.go

```diff
@@ -72,13 +72,17 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(0)
 	}
 
-	r.Reporter.Debugf("Deleting cluster '%s'", clusterKey)
-	cluster, err := r.OCMClient.DeleteCluster(clusterKey, r.Creator)
+	cluster, err := r.OCMClient.GetCluster(clusterKey, r.Creator)
```

Comment:
I think you are able to rely on fetchCluster instead 
https://github.com/openshift/rosa/blob/master/cmd/describe/cluster/cmd.go#L69


Not all commands are using it yet but I think we should be moving towards it